backend/routes/db.py

```diff
@@ -497,18 +516,6 @@ def _atlas_json_from_defs(defStr: List[Dict]) -> Dict:
     return atlas_json_from_defs(defs)
 
 
-@router.get('/omop-id-from-concept-name/{name}')
-def omop_id_from_concept_name(name):
```

Comment:
## Bugfix: Duplicate `omop_id_from_concept_name()` (add caching?)
@Sigfried There were 2 functions for `omop_id_from_concept_name()`, and they were exactly the same, with the exception that one of them had `@cache` on it. This uncached one was the 2nd of the two, so I'm guessing the prior one was disregarded. This means that previously this was not caching, but now it will. I think this is intended though.